packages/frontend/src/components/accounts/batch_import_accounts/index.js

```diff
@@ -75,6 +179,7 @@ const BatchImportAccounts = ({ accountIdToKeyMap, onCancel }) => {
         return <BatchImportAccountsSuccessScreen accounts={accountsApproved} />;
     }
 
+    const accounts = state.accounts.filter(({ accountId }) => Boolean(accountId));
```

Comment:
@loosefer Why is this necessary? If it is can we use a memoized version?

Reply:
@esaminu Legacy from my firsts commits, deleted